pkg/kubelet/stats/cri_stats_provider_test.go

```diff
@@ -421,87 +421,89 @@ func TestCRIListPodCPUAndMemoryStats(t *testing.T) {
 	)
 
 	stats, err := provider.ListPodCPUAndMemoryStats()
-	assert := assert.New(t)
-	assert.NoError(err)
-	assert.Equal(4, len(stats))
+	assertObject := assert.New(t)
```

Comment:
curious, why to rename the variable?

Reply:
this definitely makes review harder. Like this line wasn't changed, but still highlighted:

![image](https://user-images.githubusercontent.com/9950081/92861582-889bea80-f3ae-11ea-9c75-7c495327f73d.png)